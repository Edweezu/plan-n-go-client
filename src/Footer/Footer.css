.fas {
    margin-right: 3px;
}

a {
    color: rgb(0, 0, 238);
}

.contact-item {
    display: inline-block;
    padding-right: 0.8rem;
    margin: 0 auto;
}

/* .contact-item a {
    color: white;
} */

.mail {
    padding-left: 0.9rem;
}

/* .copyright {
    text-align: center;
} */

.copyright p {
    font-size: 0.7rem;
}

footer {
    /* text-align :center; */
    width: 100%;
    /* background-color: #272833;
    color: white; */
    font-weight: 500;
    margin-top: 4rem;
    /* padding: 0.5rem 1rem; */
    line-height: 0.6rem;
}

footer ul {
    margin-block-start: 0em;
    margin-block-end: 0rem;
    padding: 0px;
}

li {
    list-style: none
}