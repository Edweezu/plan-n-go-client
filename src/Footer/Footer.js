import React from 'react'
import './Footer.css'

export default class Footer extends React.Component {
    render () {
        return (
            <footer>
                <ul>
                    <li className="copyright">
                        <p><i className="fas fa-copyright"></i>2019 Edwin Qiu</p>
                    </li>
                    <li className="contact-item mail">
                            <a href="mailto:edwinqiu93@gmail.com?subject=Hi" target="_blank" aria-label="Email Edwin Qiu" rel='noopener noreferrer'>
                                <i className="far fa-envelope" title="Email"></i>
                            </a>
                    </li>
                    <li className="contact-item">
                        <a href="https://www.linkedin.com/in/edwinqiu/" target="_blank" aria-label="Edwin Qiu's Linkedin Profile" rel='noopener noreferrer'>
                            <i className="fab fa-linkedin" title="Linkedin"></i>
                        </a>
                    </li>
                    <li className="contact-item">
                        <a href="https://github.com/Edweezu" target="_blank" aria-label="Edwin Qiu's Github Profile" rel='noopener noreferrer'>
                            <i className="fab fa-github" title="Github"></i>
                        </a>
                    </li>
                </ul>
            </footer>
        )
    }
}