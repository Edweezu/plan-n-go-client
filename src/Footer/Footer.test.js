import React from 'react'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import Footer from './Footer'


describe('Footer component', () => {
    it('renders the given component', () => {
        const wrapper = shallow(<Footer />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})