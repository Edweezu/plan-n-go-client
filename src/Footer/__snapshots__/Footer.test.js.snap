// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders the given component 1`] = `
<footer>
  <ul>
    <li
      className="copyright"
    >
      <p>
        <i
          className="fas fa-copyright"
        />
        2019 Edwin Qiu
      </p>
    </li>
    <li
      className="contact-item mail"
    >
      <a
        aria-label="Email Edwin Qiu"
        href="mailto:edwinqiu93@gmail.com?subject=Hi"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="far fa-envelope"
          title="Email"
        />
      </a>
    </li>
    <li
      className="contact-item"
    >
      <a
        aria-label="Edwin Qiu's Linkedin Profile"
        href="https://www.linkedin.com/in/edwinqiu/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fab fa-linkedin"
          title="Linkedin"
        />
      </a>
    </li>
    <li
      className="contact-item"
    >
      <a
        aria-label="Edwin Qiu's Github Profile"
        href="https://github.com/Edweezu"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fab fa-github"
          title="Github"
        />
      </a>
    </li>
  </ul>
</footer>
`;
